docs/swagger.json

```diff
@@ -1,532 +1,544 @@
-{    
-    "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
-  "schemes": [
-    "https"
-  ],
-  "swagger": "2.0",
+{
+  "openapi": "3.0.1",
   "info": {
-    "description": "The purpose of Device Manager is to provide a wrapper API to manage\nDevices registered in Lamassu PKI.",
     "title": "Lamassu-Device Manager",
+    "description": "The purpose of Device Manager is to provide a wrapper API to manage\nDevices registered in Lamassu PKI.",
     "license": {
       "name": "MPL-2.0",
       "url": "https://opensource.org/licenses/MPL-2.0"
     },
     "version": "0.0.1"
   },
-  "basePath": "/v1",
+  "servers": [
+    {
+      "url": "https://jon.io:8089/v1"
```

Comment:
Please use https://dev.lamassu.io:8089/v1 as url

Reply:
In the last commit ("Swagger Auth OpenApi 3.0") are all changes made.